changelogs/unreleased/gh-6597-update-luarocks.md

```diff
@@ -1,3 +1,3 @@
-## feature/misc
+## bugfix/misc
 
 * Updated luarocks version to 3.9.2 (gh-6597).
```

Comment:
Same here: it was backported to 2.11 in order to fix some problem and we should have a problem statement in the changelog entry for a bugfix.

Reply:
fixed